src/runtime_src/core/edge/drm/zocl/sched_exec.c

```diff
@@ -1468,10 +1468,8 @@ copy_sk_return(struct sched_cmd *cmd)
 	scu = sk->sk_cu[cmd->cu_idx];
 	cu_regfile = scu->sc_vregs;
 
-	/* Reusing regmap for return code */
-	/* In future consider adding dedicated field for return code in ert_start_kernel_cmd struct */
-	if (skc->count > 1)
-		skc->data[skc->extra_cu_masks] = cu_regfile[1];
+	/* payload used for return code; payload & cu_mask are not used during cmd completion steps*/
```

Comment:
1) though it is not strongly enforced, it is better, when possible, to follow the 80 column limitation (https://www.kernel.org/doc/html/v4.10/process/coding-style.html#breaking-long-lines-and-strings). 
2) please add a space before '*/", i.e. "steps */"

Reply:
done